### DISCUSSION TITLE
running two proxy servers

### DISCUSSION THREAD

**Discussion by t2clej** [NginxProxyManager/nginx-proxy-manager#1444] [Q&A]
I have nginx proxy server running on Synology NAS - works well. I was going to setup another nginx proxy server on my other NAS as backup in case one of the NAS went down. If I do this, how do I handle the ports? 80 and 443 are already forwarded to the first NAS. Thanks.

**Comment by varunsridharan** [comment]
You need to manually change the IP Address in the router's port forwarding page
